app/controllers/start/hello.js

```diff
@@ -5,4 +5,19 @@ export default Ember.Controller.extend(OnboardingControllerMixin, {
   firstNameIsEmpty: Ember.computed.empty('model.firstName'),
   lastNameIsEmpty: Ember.computed.empty('model.lastName'),
   usersNameIsEmpty: Ember.computed.or('firstNameIsEmpty', 'lastNameIsEmpty'),
+
+  actions: {
+    /**
+    * Action that transitions to next route when user hits `enter` key
+    * if firstName and lastName fields are populated
+    *
+    * @method attemptToContinue
+    */
+    attemptToContinue() {
+      let userHasName = !this.get('usersNameIsEmpty');
+      if (userHasName) {
```

Comment:
I think this might be a bit cleaner
```
let usersNameIsEmpty = this.get('usersNameIsEmpty');

if (!usersNameIsEmpty) {
```

or even adding a computed property `userHasName: Ember.computed.not('usersNameIsEmpty'),`
and then 

```
let userHasName = this.get('userHasName');

if (userHasName) {
```
is a bit more embery